inspirehep/modules/editor/views.py

```diff
@@ -42,6 +51,59 @@
                       url_prefix='/editor',)
 
 
+@blueprint.route('/revisions/<rec_uuid>/revert', methods=['PUT'])
```

Comment:
using `rec_uuid` here and `recid` in another route makes it a bit weird but avoids

```
pid = PersistentIdentifier.query.filter_by(pid_value=recid).one()
record = get_db_record(pid.pid_type, recid)
```

and uses more simple

```record = Record.get_record(rec_uuid)```

Reply:
but also because of this, we have to return, `rec_uuid` from `get_revisions` method.